codex_of_the_damned/templates/archetypes/bibliodeque/anson-grooming.html

```diff
@@ -15,21 +15,77 @@ <h1><img src="{{ url_for('static', filename='img/icons/clan-toreador.svg') }}">
 
 {% block archetype %}
 {% trans trimmed
-
+    anson=card("Anson"),
+    tatiana=card("Tatiana Romanov"),
+    voter_cap=card("Voter Captivation"),
+    krc=card("Kine Resources Contested"),
+    tgb=card("Toreador Grand Ball"),
+    anthelios=card("Anthelios, The Red Star"),
+    temptation=card("Temptation of Greater Power"),
+    pentex_subversion=card("Pentex™ Subversion"),
+    parity_shift=card("Parity Shift"),
+	toreador_justicar=card("Toreador Justicar"),
+    elder_library=card("Elder Library"),
+    bloat=link("/strategy/bloat", _("bloat")),
+    vote=link("/strategy/archetypes", _("Vote"), _anchor="vote"),
+    politics=link("/strategy/fundamentals", _("Political Actions"), _anchor="politics"),
+    inaki_jimenez_deck=link("/deck-search", _("Iñaki Jimenez's deck"), twda_id="2k9fsbarakaldolate")
 %}
 <h2>Highlights</h2>
 <p>
-
+    This is a breed and boon deck starring {{ anson }} and uses  {{ pre }} as the
+	discipline to push votes. The purpose is to have him out as fast as possible
+	to use {{ politics }} to kill your prey.
 </p>
 
 <h2>Tips & Tricks</h2>
 <p>
+    Using {{ grooming }} cost quite an amount of blood and it is necessary to
+	replenish your vampire from time to time. Use {{ voter_cap }} to do this
+	instead of hunting as much as you can. This way you won't lose any action
```

Comment:
remove "you". One, or reword the sentence to make it neutral vis-a-vis the reader